try:
    pass
    print(hi)
except Exception as e:
    pass
    print(f"we got an exception - {e}")
    #raise
print("hello")

# else:
#     pass
# finally:
#     pass